### PR TITLE
Configure CSP_INCLUDE_NONCE_IN env variable for Exporter FE

### PR DESCRIPTION
**Pull request by saruniitr**

### CODE REVIEW

**Review by r4vi — DISMISSED** [review]

File: conf/base.py [hunk 1]
```diff
@@ -184,6 +184,7 @@
 CSP_SCRIPT_SRC = env.tuple("CSP_SCRIPT_SRC", default=("'self'",))
 CSP_FONT_SRC = env.tuple("CSP_FONT_SRC", default=("'self'",))
 CSP_REPORT_ONLY = env.bool("CSP_REPORT_ONLY", False)
+CSP_INCLUDE_NONCE_IN = env.tuple("CSP_INCLUDE_NONCE_IN", default=("'script-src'",))
```
Comment (r4vi) [inline conf/base.py:187]: I don't think it needs to be single quoted like `'self'` the env var that's set on exporter is currently just `CSP_INCLUDE_NONCE_IN="script-src,"`

Comment (r4vi) [reply conf/base.py:187]: https://github.com/mozilla/django-csp/blob/53278b9946a8c23c07ca3ca53d4338f4ad48068f/docs/configuration.rst#changing-the-policy

Comment (r4vi) [inline conf/base.py:187]:
```suggestion
CSP_INCLUDE_NONCE_IN = env.tuple("CSP_INCLUDE_NONCE_IN", default=("script-src",))
```

**Review by r4vi — CHANGES_REQUESTED** [review]

File: conf/base.py (hunk 1 above)
Comment (saruniitr) [reply conf/base.py]: ok, marking it as resolved as updated changes are pushed

**Review by r4vi — APPROVED** [review]

**Pull request merged**